terraform/network.tf

```diff
@@ -1,16 +1,16 @@
 # VPC module for setting up vpc
 module "vpc" {
-  source                     = "github.com/azavea/terraform-aws-vpc?ref=3.1.1"
+  source                     = "github.com/azavea/terraform-aws-vpc?ref=5.0.1"
```

Comment:
This upgrade results in the following changes:

```
  + module.vpc.aws_network_interface_sg_attachment.bastion
      id:                                        <computed>
      network_interface_id:                      "eni-094359129c2174062"
      security_group_id:                         "sg-1b3e7965"
```

Additionally, because we created a new bastion instance, its Route 53 record will be updated:

```
  ~ aws_route53_record.bastion
      allow_overwrite:                           "" => "true"
      records.1042293117:                        "" => "<truncated>"
      records.2771329930:                        "<truncated>" => ""
```